options.html

```diff
@@ -99,6 +99,11 @@ <h2>
                 <div id="video_loader_settings">
                     <input type="checkbox" id="video_loader_hd"/><label for="video_loader_hd">Show in HD (when available)</label>
                 </div>
+                <p>
+                    <input type="checkbox" class="script_check" id="embed_socials" />
+                    <label for="video_loader">Embed Social Links</label>
+                </p>
+                <p class="indent info">Replace links to Twitter and Instagram with embedded posts (note: disable tracker blocking).</p>
```

Comment:
Can you expand the options text to explain what "disable tracker blocking" means?  If it's required, it would be nice if it had a link to some instructions on how to do it in Chrome and Firefox.  I wasn't sure what this was asking me to do.

Reply:
Sure. Basically, Firefox optionally lets you prevent javascript and cookie trackers from running, and that conflicts with Twitter/Instagram embedding. I don't think Chrome does this at the moment unless you're using an extension like Ghostery or some such.

I'll add a more detailed blurb about it in the feature option on the options page.